### DISCUSSION TITLE
ADD ABILITY TO DISABLE FLOW OPENING WITH WIN+R

### DISCUSSION THREAD

**Discussion by sammyjsin** [Flow-Launcher/Flow.Launcher#1557] [Q&A] [answered]
Please Add the option so I can use the default Run box by using Win+R, Its not currently in settings and Flow cant always do all commands that the run box can.

**Comment by onesounds** [comment, marked as answer]
check this part in Shell Plugin. (Replace Win+R)
![image](https://user-images.githubusercontent.com/6903107/202836294-64badf7a-4782-4a02-83b1-ae1b7dd1430c.png)

**Reply by sammyjsin** [reply]
THANK YOU
